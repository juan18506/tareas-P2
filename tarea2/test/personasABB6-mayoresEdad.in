crearArbolPersonas

# arbol vacio, retorna arbol vacio
# mayoresEdadArbolPersonas 3

crearAgenda
crearPersona 5 55 Persona5
crearFecha 5/55/55
crearEvento 5 Descripcion5
agregarEventoPersona
insertarPersonaArbolPersonas

# arbol con 1 elemento - maxId 5
maxIdPersonaArbolPersonas

crearAgenda
crearPersona 10 100 Persona10
insertarPersonaArbolPersonas

crearAgenda
crearPersona 2 22 Persona2
insertarPersonaArbolPersonas

# arbol con 3 elementos max Id 10
maxIdPersonaArbolPersonas

# arbol con mayores a 30 - 2 elementos
mayoresEdadArbolPersonas 30

crearAgenda
crearPersona 20 200 Persona20
insertarPersonaArbolPersonas

crearAgenda
crearPersona 7 77 Persona7
insertarPersonaArbolPersonas

crearAgenda
crearPersona 1 11 Persona1
insertarPersonaArbolPersonas

crearAgenda
crearPersona 3 33 Persona3
insertarPersonaArbolPersonas

# arbol con mayores a 35 - 4 elementos
mayoresEdadArbolPersonas 35

# remuevo la persona 20
removerArbolPersonas 20

# remuevo la persona 5
removerArbolPersonas 5

# remuevo la persona 2
removerArbolPersonas 2

crearAgenda
crearPersona 4 44 Persona4
# inserto la persona 4
insertarPersonaArbolPersonas

# arbol con mayores a 35 - 3 elementos
mayoresEdadArbolPersonas 35

liberarArbolPersonas

Fin
