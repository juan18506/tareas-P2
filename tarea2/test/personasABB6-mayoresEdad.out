1>2>#  arbol vacio, retorna arbol vacio.
3>#  mayoresEdadArbolPersonas 3.
4>5>6>7>8>9>10>#  arbol con 1 elemento - maxId 5.
11>La persona con mayor id en el árbol es: 
Persona 5:  Persona5, 55 años
Evento 5:  Descripcion5
Fecha: 5/55/55
12>13>14>15>16>17>18>#  arbol con 3 elementos max Id 10.
19>La persona con mayor id en el árbol es: 
Persona 10:  Persona10, 100 años
20>#  arbol con mayores a 30 - 2 elementos.
21>Persona 5:  Persona5, 55 años
Evento 5:  Descripcion5
Fecha: 5/55/55
Persona 10:  Persona10, 100 años
22>23>24>25>26>27>28>29>30>31>32>33>34>#  arbol con mayores a 35 - 4 elementos.
35>Persona 5:  Persona5, 55 años
Evento 5:  Descripcion5
Fecha: 5/55/55
Persona 7:  Persona7, 77 años
Persona 10:  Persona10, 100 años
Persona 20:  Persona20, 200 años
36>#  remuevo la persona 20.
37>38>#  remuevo la persona 5.
39>40>#  remuevo la persona 2.
41>42>43>44>#  inserto la persona 4.
45>46>#  arbol con mayores a 35 - 3 elementos.
47>Persona 4:  Persona4, 44 años
Persona 7:  Persona7, 77 años
Persona 10:  Persona10, 100 años
48>49>Fin.
